howso/analysis.amlg

```diff
@@ -1,6 +1,7 @@
 ;Contains methods for hyperparameter analysis and feature deviation and weights calculations.
 (null
 	;automatically analyze the model using stored parameters from previous analyze calls
+	;{long_running (true) idempotent (false)}
```

Comment:
read_only (false) as well

Reply:
switched default to false so it should be ok